marshaller.go

```diff
@@ -589,6 +768,21 @@ func marshal(reflectedValue reflect.Value, reflectedType reflect.Type, childRela
 				buff.WriteString(subStr)
 			}
 		}
+		if orderByClausePresent {
+			relationName := ""
+			if childRelationName != "" {
```

Comment:
We will need to add test for order by in child tables. If not supported, we should explicitly disallow orderBy clause for nested queries.

Reply:
Added unit test in a followup commit